### PR TITLE
upcoming: [DI-21118] - Monitor scope as part of PAT token

### PR DESCRIPTION
**Pull request by ankita-akamai**
## Description 📝
A new 'Monitor' scope is enabled within Cloud Manager UI for PAT token creation.

## Changes  🔄
- Added new scope 'Monitor' for PAT token creation.

## Target release date 🗓️
<TBD>

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Before | After |

## How to test 🧪

1. Login as mock user.
2. Click on "mock-user" profile tab at the right top.
3. Go to api-tokens, click on "create a personal access token", you should see "Monitor" listed as one of the scopes with freedom of selecting any/all three types of access.

## As an Author I have considered 🤔

*Check all that apply*

- [x] Use React components instead of HTML Tags
- [x] Proper naming conventions like cameCase for variables & Function & snake_case for constants
- [x] Use appropriate types & avoid using "any"
- [x] No type casting & non-null assertions
- [ ] Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [x] Providing/Improving test coverage
- [x] Use sx props to pass styles instead of style prop
- [x] Add JSDoc comments for interface properties & functions
- [x] Use strict equality (===) instead of double equal (==)
- [x] Use of named arguments (interfaces) if function argument list exceeds size 2
- [x] Destructure the props
- [x] Keep component size small & move big computing functions to separate utility
- [x] 📱 Providing mobile support

### CODE REVIEW

**Review by nikhagra-akamai — APPROVED** [review]

File: packages/manager/src/features/Profile/APITokens/utils.test.ts [hunk 1]
```diff
@@ -304,9 +315,10 @@ describe('APIToken utils', () => {
         ];
         expect(allScopesAreTheSame(scopes)).toBe(2);
       });
-      it('should return null if all scopes are different', () => {
+      it('should return null if all scopes are not same', () => {
```
Comment (venkymano-akamai) [inline packages/manager/src/features/Profile/APITokens/utils.test.ts]: do we need change the name here?, is it even required for an already existing message

Comment (ankita-akamai) [reply packages/manager/src/features/Profile/APITokens/utils.test.ts]: It is very confusing, it says all scopes are different which is wrong literally. It should be "not same".

File: packages/manager/src/features/Profile/APITokens/utils.ts [hunk 2]
```diff
@@ -27,6 +28,7 @@ export const basePerms = [
 
 export const basePermNameMap = {
   account: 'Account',
+  aclp: 'ACLP',
```
Comment (venkymano-akamai) [inline packages/manager/src/features/Profile/APITokens/utils.ts]: this should be aclp or monitor or cloudpulse, needs some confirmation isn't it.

Comment (kmuddapo) [reply packages/manager/src/features/Profile/APITokens/utils.ts]: As per the latest discussion, lets use this as monitor:'Monitor'.

File: packages/manager/src/features/Profile/APITokens/utils.test.ts (hunk 1 above)
Comment (kmuddapo) [reply packages/manager/src/features/Profile/APITokens/utils.test.ts]: If its existing message, better not to change.

File: packages/manager/src/features/Profile/APITokens/ViewAPITokenDrawer.test.tsx [hunk 3]
```diff
@@ -111,13 +111,14 @@ describe('View API Token Drawer', () => {
         {...props}
         token={appTokenFactory.build({
           scopes:
-            'databases:read_only domains:read_write child_account:read_write events:read_write firewall:read_write images:read_write ips:read_write linodes:read_only lke:read_only longview:read_write nodebalancers:read_write object_storage:read_only stackscripts:read_write volumes:read_only vpc:read_write',
+            'aclp:read_write databases:read_only domains:read_write child_account:read_write events:read_write firewall:read_write images:read_write ips:read_write linodes:read_only lke:read_only longview:read_write nodebalancers:read_write object_storage:read_only stackscripts:read_write volumes:read_only vpc:read_write',
```
Comment (kmuddapo) [inline packages/manager/src/features/Profile/APITokens/ViewAPITokenDrawer.test.tsx]: how about read_only for aclp/monitor even though its a test?

**Review by venkymano-akamai — APPROVED** [review]

**Pull request merged**